src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -1,21 +1,97 @@
-(function (global, doc) {
+(function (global, doc, bootstrap, Translator) {
+    let toggleAllTimeout;
+    const toggleAllBtns = [...doc.querySelectorAll(`[data-multi-collapse-btn-id]`)];
+    const toggleMultiCollapseBtn = (btn, changeToCollapseAll) => {
+        const displayedText = changeToCollapseAll
```

Comment:
I will prefer create 2 variables and move them from this function next in `toggleMultiCollapseBtn()` just toggle them, but this is not must have :) 
```suggestion
        const collapseAllLabel = Translator.trans(/*@Desc("Collapse all)*/ 'product_type.edit.section.attribute_collapse_all', {}, 'ibexa_product_catalog');
        const expandAllLabel = Translator.trans( /*@Desc("Expand all)*/ 'product_type.edit.section.attribute_expand_all', {}, 'ibexa_product_catalog');
```